### DISCUSSION TITLE
PROBLEM

### DISCUSSION THREAD

**Discussion by supperzaki** [nyxiereal/ReVancedPacker#20] [General]
i have this error when  i reach the part of creating apk i also ensure to install java that you provided in the installer also python 
and thanks.

![LD](https://user-images.githubusercontent.com/98193760/179393222-1c0a134d-1d6d-4edf-8cdf-461abd99cf87.png)

**Comment by nyxiereal** [comment]
did you use "include all" option?

**Reply by supperzaki** [reply]
Thanks for you response 
If you mean ((use all integration or include selected integrations)) 
I select ((2|exclude selected )

**Reply by nyxiereal** [reply]
do you have java with add to path enabled?

**Reply by supperzaki** [reply]
Yes and used this video as a guid 
https://youtu.be/rxKDcenUJhc

**Reply by nyxiereal** [reply]
NONONONO, while setting up java 17!
